lib/const.rb

```diff
@@ -133,6 +133,7 @@
 CREW_NOT_LINKS                       = ENV.fetch('CREW_NOT_LINKS', '0').eql?('1')                       # or use no_links
 CREW_NOT_STRIP                       = ENV.fetch('CREW_NOT_STRIP', '0').eql?('1')                       # or use no_strip
 CREW_NOT_SHRINK_ARCHIVE              = ENV.fetch('CREW_NOT_SHRINK_ARCHIVE', '0').eql?('1')              # or use no_shrink
+CREW_SOURCE_BASHRC_ENABLED           = ENV.fetch('CREW_SOURCE_BASHRC_ENABLED', '0').eql?('1')
```

Comment:
If we're not using a property, why do we even need a constant?

Reply:
It is being used in the postinstall:
```ruby
def post_install
  if @pkg.print_source_bashrc? || @pkg.gnome?
    crewlog "@pkg.print_source_bashrc?:#{@pkg.print_source_bashrc?} @pkg.gnome?:#{@pkg.gnome?}"
    # Check to see if the env variable was set, because without the trap
    # set, USR1 will kill crosh.
    if CREW_SOURCE_BASHRC_ENABLED
      Process.kill('USR1', Process.ppid)
      crewlog '~/.bashrc was automatically sourced.'
    else
      at_exit do
        exec 'bash'
      end
    end
  end

  GnomePostinstall.add @pkg.name if @pkg.gnome?

  # return unless the postinstall function was defined by the package recipe
  return unless @pkg.method(:postinstall).source_location[0].include?("#{@pkg.name}.rb")

  Dir.mktmpdir do |post_install_tempdir|
    Dir.chdir post_install_tempdir do
      puts "Performing post-install for #{@pkg.name}...".lightblue
      @pkg.postinstall
    end
  end
end
```

Reply:
I will move it out of that section though...

Reply:
So if you disable it, how does the user know they need to execute `source ~/.bashrc` manually?

Reply:
Variable renamed to `CREW_SOURCE_BASHRC_ACTIVATED` to be used for when `crew_profile_base` is upgraded, and the new USR1 trap isn't set yet because the new profile hasn't been loaded yet.